App.js

```diff
@@ -1,26 +1,71 @@
-import React from "react"
-import styled from "styled-components/native"
-
-const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
-
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
-
-export default App
+import React, { useState, useEffect } from "react";
+import { SafeAreaView } from "react-native";
+import styled from "styled-components/native";
+import Header from "./assets/components/Header";
+import Title from "./assets/components/Title";
+import CustomButton from "./assets/components/CustomButton";
+import LoadingPokemonCard from "./assets/components/LoadingPokemonCard";
+import PokemonCard from "./assets/components/PokemonCard";
+
+const randomizePokemon = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min); // max value is exclusive and min value inclusive, so the value is no lower than min but less than and not equal to max
+};

```

Comment:
nice randomizer.
This could even be extracted into its own file, and made general to be used in other projects.
Then just import it here and use it as you do on line 18